sdk/servicebus/azure-servicebus/README.md

```diff
@@ -369,6 +369,24 @@ a generator-style receive will run for before exiting if there are no messages.
 
 ### Common Exceptions
 
+The Service Bus APIs generate the following exceptions in azure.servicebus.exceptions:
```

Comment:
The one callout I'd make here, if I put on my "user trying to figure out the service" hat; is that I might have notes for e.g. "this can be retried" or "this means you must do X"

This is covered sufficiently in some places, e.g. MessageLockExpired.  But other places leave questions for me, e.g. NoActiveSession, or ServiceBusAuthenticationError. (why should I know the former is retriable, but the latter isn't?)

LMK if this resonates with you or if you think I'm overloading this block, and we can find other ways to capture this.